src/daemon/visor.go

```diff
@@ -116,7 +116,9 @@ func (vs *Visor) Run() error {
 			return err
 		case req := <-vs.reqC:
 			func() {
-				req()
+				cxt, cancel := context.WithDeadline(context.Background(), time.Now().Add(time.Minute))
```

Comment:
I thought context deadline exiting was unsafe?

Reply:
Yes, hmm, might have thread safe problem, will remove this, lol